ui/src/app/components/help-sidebar-icons.tsx

```diff
@@ -112,25 +113,42 @@ export interface IconConfig {
 
 const displayRuntimeStatusIcon = (
   icon: IconConfig,
-  workspaceNamespace: string,
-  featureFlaggedStyling: boolean = false
+  workspaceNamespace: string
 ) => {
-  // We always want to show the thunderstorm icon.
-  // For most runtime statuses (Deleting and Unknown currently excepted), we will show a small
-  // overlay icon in the bottom right of the tab showing the runtime status.
-  return (
-    <FlexRow
-      style={{
+  const { enableGkeApp } = serverConfigStore.get().config;
+
+  const jupyterAssets = appAssets.find(
```

Comment:
What is the difference between an asset and an app? The find function returns a single item, would it make sense to make this variable singular?